src/Tablebot/Plugins/Roll/Dice/DiceParsing.hs

```diff
@@ -25,26 +26,75 @@ import Tablebot.Plugins.Roll.Dice.DiceFunctions
     integerFunctions,
     listFunctions,
   )
-import Tablebot.Utility.Parser (integer, parseCommaSeparated1, skipSpace)
+import Tablebot.Utility.Parser (integer, parseCommaSeparated1, skipSpace, skipSpace1)
 import Tablebot.Utility.SmartParser (CanParse (..), (<??>))
 import Tablebot.Utility.Types (Parser)
-import Text.Megaparsec (MonadParsec (try), choice, failure, optional, (<?>), (<|>))
+import Text.Megaparsec (MonadParsec (try), choice, failure, optional, some, (<?>), (<|>))
 import Text.Megaparsec.Char (char, string)
 import Text.Megaparsec.Error (ErrorItem (Tokens))
 
 -- | An easier way to handle failure in parsers.
 failure' :: T.Text -> Set T.Text -> Parser a
 failure' s ss = failure (Just $ Tokens $ NE.fromList $ T.unpack s) (S.map (Tokens . NE.fromList . T.unpack) ss)
 
+varName :: Parser T.Text
+varName = T.pack <$> some (choice $ char <$> '_' : ['a' .. 'z'])
```

Comment:
This seems to require that vars only consist of lowercase / underscore - if so, this should be documented in Roll.md.